src/__tests__/App.test.tsx

```diff
@@ -1,9 +1,30 @@
 import React from 'react'
-import {shallow} from 'enzyme'
+import {mount, ReactWrapper} from 'enzyme'
```

Comment:
Instead of enzyme, can we try to use testing-library in this website? It is actually widely used, and the default for cra as O know